Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/view/MultiImageView.java

```diff
@@ -486,6 +488,12 @@ private void setGifFile(File file) {
 
     private void setVideo(Loadable loadable, PostImage postImage) {
         BackgroundUtils.ensureMainThread();
+
+        if (getWidth() == 0 || getHeight() == 0) {
```

Comment:
This will necessarily never be the case, as these properties are ensured to be greater than zero by waitForMeasure.

Reply:
Yeah, I though so too but then I saw these logs: https://github.com/Adamantcheese/Kuroba/issues/540#issuecomment-581014211

Reply:
I have no idea why or how this happens but it does.